index.html

```diff
@@ -0,0 +1,127 @@
+                </div>
+                <nav class="nav-bar">
+                <img src="images/christmas_2012_new_2857.jpg" alt="logo" class="Logo">
+                <ul class="navitems">
+                    <li><a href="./about.html" class="a-abt"> About</a></li>
+                    <li><a href="#"> Courses</a></li>
+                    <li><a href="#main"> Recipes</a></li>
+                    <li><a href="./about.html#fav"> Favourites</a></li>
+                    <li><a href="./about.html#head">Contact</a></li>
+                </ul>
+                <ul class="links">
+                    <li><a href="#Contact"> MyRecipes</a></li>
+                </ul>
+                </nav>
+            </nav>
+            <div class="head-page">
+                <div class="info-page">
+                    <h2 class="Greetings">"Hello! myFoodies"</h2>
+                    <h1 class="title">RECIPES TRIED BY IPTY</h1>
+                    <p class="description">I would like to share recipes that I have tried making. They have turned out to be amazing and delicious. I have an amazing journey with food and desserts and the love of learning new recipes. Sharing this with all of you is fruitful.</p>
+                    <div class="date_tag">
+                    <h3 class="date">Every Saturday with a new Recipe</h3>
+                    <a href="#" class="tag">@myRecipes</a>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </header>
+
+    <main class="main">
+        <div class="main-page" id="main">
+        <h2 class="recipes">Recipes</h2>
+        <div class="container">
+            <div class="recipe-block">
+                <img src="images/rice-bowl.png" alt="image" class="img">
+                <h3 class="category">Main</h3>
+                <p class="recipe-text">Sweet and yummy</p>
+            </div>
+            <div class="recipe-block">
+                <img src="images/cake.png" alt="image" class="img">
+                <h3 class="category">Desserts</h3>
+                <p class="recipe-text"> Sweet and yummy</p>
+            </div>
+            <div class="recipe-block">
+                <img src="images/pizza-slice.png" alt="image" class="img">
+                <h3 class="category">Snacks</h3>
+                <p class="recipe-text"> Sweet and yummy</p>
+            </div>
+            <div class="recipe-block">
+                <img src="images/pizza-slice.png" alt="image" class="img">
+                <h3 class="category">Appetizer</h3>
+                <p class="recipe-text"> Sweet and yummy</p>
+            </div>
+            <div class="recipe-block">
+                <img src="images/pizza-slice.png" alt="image" class="img">
+                <h3 class="category">Drinks</h3>
+                <p class="recipe-text"> Sweet and yummy</p>
+            </div>
+        </div>
+        <button class="recipe_btn">More Recipes</button>
+        <a href="#">See More</a>
+        </div>    
```

Comment:
- [Optional] Consider using a `<section>` element to wrap this section of your document instead of a `<div>`. This is best practice for HTML semantics.

- [ ] Please as required by the previous reviewer, kindly change the background of this section to match the design. You are to stick to design guidelines as much as possible while personalizing the contents.
 ### Your design
![y1](https://user-images.githubusercontent.com/94551484/190871609-ccc1680c-d6b0-4c15-b2b3-8fea413b2843.png)

### Design guidelines
![r1](https://user-images.githubusercontent.com/94551484/190871627-32667224-1114-4a12-8b24-2283216ff52b.png)